.github/workflows/github-release.yml

```
@@ -0,0 +1,16 @@
+    branches:
+      - main
+
+jobs:
+  setup:
+    uses: zhumeisongsong/shared-actions/.github/workflows/reusable-pnpm-setup.yml@main
+
+  release:
+    needs: setup
+    uses: zhumeisongsong/shared-actions/.github/workflows/reusable-semantic-release.yml@main
+    secrets:
+      REPO_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)